aspnetcore/includes/run-the-app.md

```diff
@@ -5,9 +5,27 @@ no-loc: [Home, Privacy, Kestrel, appsettings.json, "ASP.NET Core Identity", cook
 
 * Press Ctrl+F5 to run without the debugger.
 
-  [!INCLUDE[](~/includes/trustCertVS.md)]
+<!-- replace all of this with updated includes  -->
 
-  Visual Studio starts [IIS Express](/iis/extensions/introduction-to-iis-express/iis-express-overview) and runs the app. The address bar shows `localhost:port#` and not something like `example.com`. That's because `localhost` is the standard hostname for the local computer. Localhost only serves web requests from the local computer. When Visual Studio creates a web project, a random port is used for the web server.
+Select **RazorPagesMovie** in **Solution Explorer**, and then press Ctrl+F5 to run without the debugger.
```

Comment:
```suggestion
Press Ctrl+F5 to run without the debugger.
```

Reply:
Just a note so I can remember what happened with this request: This suggested change for run-the-app already exists in Main now so I will remove run-the-app.md from my pr.  I will also not use inlcudes/run-the-app.md for this document since the port number is not 5001 but randomly assigned for this tutorial and we are temporarily completely excluding VS for Mac tabs until VS 2022 for the mac comes out.